package avalon.network;

message GameSettings {
    required int32 players = 1;
    required bool merlin = 2 [ default = false ];
    required bool percival = 3 [ default = false ];
    required bool mordred = 4 [ default = false ];
    required bool morgana = 5 [ default = false ];
    required bool assassin = 6 [ default = false ];
    required bool oberon = 7 [ default = false ];
    required int32 client = 8;
    required int32 evil_count = 9;
    required int32 vote_track_len = 10;
    required int32 quest_track_len = 11;
    repeated int32 players_per_quest = 12;
    repeated int32 fails_per_quest = 13;
}
